crypto/x509/t_x509.c

```diff
@@ -302,25 +302,28 @@ int X509_signature_print(BIO *bp, const X509_ALGOR *sigalg,
                          const ASN1_STRING *sig)
 {
     int sig_nid;
-    if (BIO_puts(bp, "    Signature Algorithm: ") <= 0)
+    int indent = 4;
```

Comment:
Perhaps this should be const? In any case please add a blank line between the declarations and the rest of the code.

Reply:
Yes - done.